### PR TITLE
Add content table

### PR DESCRIPTION
**Pull request by taxmeifyoucan**
README got pretty long and basic content table would help users to navigate.

### CODE REVIEW

**Review by MaxHillebrand — APPROVED** [review]
ACK  448497d

**Comment by Kixunil** [comment]
There's one in the upper-left corner, it was [added 8 days ago](https://github.com/isaacs/github/issues/215#issuecomment-807688648)

I do think that having something more visible is nice but we will have to keep this in sync. Maybe simple "Check upper-left corner of README for ToC?" What do you think?

**Comment by taxmeifyoucan** [comment]
@Kixunil Well, even I haven't noticed it :D I see it is github feature, this way it is not dependent on software reading md file

**Comment by MaxHillebrand** [comment]
woah, that automated ToC is super useful actually, didn't know this got added, nice!

**Comment by Kixunil** [comment]
How about having a smaller table with "featured topics"? I would consider these topics featured:

* How to setup
* Bug reporting
* Documentation
* Development

And perhaps leave a note about GH feature for the upcoming few months until people learn about it.

**Comment by Kixunil** [comment]
@taxmeifyoucan do you intend to update this PR based on my suggestion (smaller table and a hint)?

File: README.md [hunk 1]
```diff
@@ -35,7 +42,7 @@ To use the produced repository you need to also setup Microsoft dotnet repositor
 and don't forget to [verify fingerprints](https://gist.github.com/Kixunil/fb7300edcb8a8afc95e6b7b727b31f0d).
 
 ```
-gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys 3D9E81D3CA76CDCBE768C4B4DC6B4F8E60B8CF4C
+gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys 3D9E81D3CA76CDCBE768C4B4DC6B4F8E60B8CF4C
```
Comment (Kixunil) [inline README.md:45]: Please remove accidental space

**Review by Kixunil — CHANGES_REQUESTED** [review]

**Review by Kixunil — APPROVED** [review]
Thanks!

**Pull request merged**